### PR TITLE
chore: [PE-535] Adapted the FooterWithButtons into CGN screens

### PR DESCRIPTION
**Pull request by Hantex9**
## Short description
This PR adapts the CGN screens with the new `FooterWithButtons` released into the new version of the design system `v1.21.0`

## List of changes proposed in this pull request
- Removed the `SafeAreaView` from CGN screens with a `FooterWithButtons` inside

## How to test
- Run all the CGN flows such as the activation and details, you should be able to see the right footer with buttons using the right margin.

## Preview
|Before|After|
|-|-|
|  |  |

## Acknowledgment
@dmnplb

### CODE REVIEW

**Review by mastro993 — APPROVED** [review]

**Comment by dmnplb** [comment]
@Hantex9 Can we approve this PR? I'm working on removing the legacy `FooterWithButtons` in the following draft PR:
* https://github.com/pagopa/io-app/pull/5558

It would be nice not to work on this flow twice.

File: ts/features/bonus/cgn/screens/activation/CgnActivationCompletedScreen.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+
 import { cgnActivationComplete } from "../../store/actions/activation";
```
Comment (CrisTofani) [inline ts/features/bonus/cgn/screens/activation/CgnActivationCompletedScreen.tsx:3]:
```suggestion
import { cgnActivationComplete } from "../../store/actions/activation";
```

File: ts/features/bonus/cgn/screens/activation/CgnActivationCompletedScreen.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { OperationResultScreenContent } from "../../../../../components/screens/
  * Screen which is displayed when a user requested a CGN activation
  * and it has been correctly activated
  */
+
 const CgnActivationCompletedScreen = () => {
```
Comment (CrisTofani) [inline ts/features/bonus/cgn/screens/activation/CgnActivationCompletedScreen.tsx:12]:
```suggestion
const CgnActivationCompletedScreen = () => {
```

File: ts/features/bonus/cgn/screens/activation/CgnActivationIneligibleScreen.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+
 import { cgnActivationCancel } from "../../store/actions/activation";
```
Comment (CrisTofani) [inline ts/features/bonus/cgn/screens/activation/CgnActivationIneligibleScreen.tsx:3]:
```suggestion
import { cgnActivationCancel } from "../../store/actions/activation";
```

**Comment by CrisTofani** [comment]
Closing this since the refactoring is part of #5651 and #5617

**Pull request closed without merging**